components/engine/engine-odata/src/main/java/org/eclipse/dirigible/components/odata/transformers/ODataDatabaseMetadataUtil.java

```diff
@@ -99,6 +99,7 @@ public class ODataDatabaseMetadataUtil {
         SQL_TO_ODATA_EDM_TYPES.put("DECIMAL", "Edm.Decimal");
         SQL_TO_ODATA_EDM_TYPES.put("REAL", "Edm.Single");
         SQL_TO_ODATA_EDM_TYPES.put("FLOAT", "Edm.Single");
+		SQL_TO_ODATA_EDM_TYPES.put("FLOAT8", "Edm.Double");
```

Comment:
Please, include also what is missed from STRING_TO_DATABASE_TYPE in DataTypeUtils